### ISSUE TITLE
Drag Ref inside the drop

### ISSUE THREAD

**Issue by sheikharifulislam** [react-dnd/react-dnd#3633]
I want to access Drag ref inside the drop. is possible this?

**Comment by sachin36987** [comment]
Simply use global state library to access the values.

**Comment by sheikharifulislam** [comment]
> Simply use global state library to access the values.

i mean I want to access. ref.current.

**Comment by sachin36987** [comment]
yes, that's what I am saying, that you can store the ref inside the global state and subscribe to that state in drop component. Also, since you may not want to have any reactivity then a simple object or context will also suffice.
